inc/thorvg.h

```diff
@@ -977,6 +977,19 @@ class TVG_EXPORT Shape final : public Paint
      */
     Result fill(FillRule r) noexcept;
 
+    /**
+     * @brief Sets the rendering order of the fill and stroke.
+     *
+     * @param[in] fill Number specifying the order in which a fill will be rendered [0 ~ 1]. Rendered first by default (0).
+     * @param[in] stroke number specifying the order in which a stroke will be rendered [0 ~ 1]. Rendered second by default (1).
+     *
+     * @retval Result::Success When succeed.
+     * @retval Result::InvalidArguments If any of the arguments has the same value as another or is outside the range 0-1.
+     *
+     * @BETA_API
+     */
+    Result order(uint8_t fill, uint8_t stroke) noexcept;
```

Comment:
I think multiple parameters are ambugious to use. even the value's range.

Let's simplify this. How about this?

Result stroke(bool above == true);    //stroke order. if true, draw stroke above the shape fill. otherwise, draw stroke under the shape fill.

Reply:
First, I think when ideas are conflicted, I'd like to recommend you to consensus first then code.
Second, float / bool are obviously different types. any issues of it?

Reply:
So I asked you a question an actual issue of it.
Were there any complie error? warning? what problems would occur when you call it?

Reply:
sorry, I didn't fully understand what you wrote.
yes, calling stroke(4) will cause an error: ```error: call to member function 'stroke' is ambiguous```
if we can live with that, I'll remove the enum and write as you suggested, stroke(bool).

Reply:
ok, thanks for info. When i tested it with msvc it didn't have any issues but gcc has different result.

//fill above stroke. 
So... What do you think?
Result Shape::fill(bool above) noexcept;

Reply:
@hermet please check now. I applied changes